cypress/fixtures/rbac.json

```diff
@@ -26,7 +27,8 @@
         "Analyze": true,
         "Upload binary": true,
         "Top action menu": {
-            "Not available": true,
+            "Not available": false,
```

Comment:
Top action menu shouldn't be available for migrators.

Reply:
Restored